package.json

```diff
@@ -8,7 +8,7 @@
     "cli": "ts-node --require ./app/test/globals.ts --require ./app/src/cli/dev-commands-global.ts app/src/cli/main.ts",
     "check:eslint": "tsc -P eslint-rules/",
     "test:eslint": "jest eslint-rules/tests/*.test.js",
-    "test:unit": "cross-env ELECTRON_RUN_AS_NODE=1 ./node_modules/.bin/electron ./node_modules/jest/bin/jest --detectOpenHandles --silent --testLocationInResults --config ./app/jest.unit.config.js",
+    "test:unit": "cross-env ELECTRON_RUN_AS_NODE=1 NODE_OPTIONS='--max_old_space_size=4096' ./node_modules/.bin/electron ./node_modules/jest/bin/jest  --detectOpenHandles --silent --testLocationInResults --config ./app/jest.unit.config.js",
```

Comment:
After electron upgrade, the cli would sometimes get a ["Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory"](https://github.com/desktop/desktop/runs/5051290822?check_suite_focus=true). Adding the node-options to increase memory allocation like we do in the `compile:prod` seems to resolve the issue.